portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -115,23 +114,40 @@ private String getParameterOrDefault(HttpServletRequest request, String name, St
 
   /*
    * Return a strictly positive integer from a url parameter, or defaultValue if not present
-   * Note: assumes defaultValue is strictly positive
+   * If defaultValue and the requested value are not positive, 1 will be returned. 
```

Comment:
If the defaultValue is not positive, you should throw an error, because it means people are using your method incorrectly.

Reply:
Noted